src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
```

Comment:
- [ ] [OPTIONAL] I would suggest to remove this test file as it is redundant for now, and you will be creating a separate test folder when you test the app in the future project :+1: